README.md

```diff
@@ -14,98 +14,78 @@ Note: If your ssh public key (`~/.ssh/id_rsa.pub`) is already on the remote mach
 
 ### Inventory
 
-Ansible manages hosts using `inventory.yml` file. Current setup has two group names:
-
-* `sentry`
-* `validator`
-
-Each group may have multiple IPs (hosts). Each Ansible command needs group name to be mentioned. Ansible runs the playbook on mentioned group's machines. Note that if you don't mention group name, Ansible will run playbook on all machines.
+Ansible manages hosts using `inventory.yml` file.
 
 **Setup inventory**
 
-Add sentry nodes IPs/hosts under the `sentry` group and add validator node's IP/host under `validator` group. 
+Add nodes ip's in `inventory.yml` as example
 
 Example:
 
 ```yml
-all:
-  hosts:
-  children:
-    sentry:
-      hosts:
-        xxx.xxx.xx.xx: # <----- Add IP for sentry node
-        xxx.xxx.xx.xx: # <----- Add IP for sentry node
-    validator:
-      hosts:
-        xxx.xxx.xx.xx: # <----- Add IP for validator node
+#add your instance ips here in palce of xx..
+xxx.xxx.xx.xx # <----- Add IP for sentry/validator node
+
 ```
 
 Note: By default the user to login is setup as ubuntu in `group_vars/all` file. If you have a specific user to be logged in with please change the username in this file.
 
 To check if nodes are reachable, run following commands:
 
 ```bash
-# to check if sentry nodes are reachable
-ansible sentry -m ping
-
-# to check if validator nodes are reachable
-ansible validator -m ping
+# to check if nodes are reachable
+ansible -m ping
```

Comment:
This would need to be `ansible all -m ping` I think

Reply:
we don't need all as we have removed groups all together form inventory so no confusion for people